components/GalleryPictures.tsx

```diff
@@ -20,15 +20,29 @@ function GalleryPictures({}: Props) {
         alt=""
       />
 
-      <div>
-        <h4>Kitchens</h4>
-        <p>From modern to traditional</p>
-        <div>
-          {/* Cabinet Style */}
-          {/* Cabinet Style */}
-          {/* Cabinet Style */}
+      <div className="px-0 md:px-10">
+        <h4 className="text-4xl font-light">Kitchens</h4>
+        <p className="font-bold text-2xl mt-1">From modern to traditional</p>
+        <div className="flex space-x-2 my-2">
+          <img
+            className="h-10 w-10 rounded-full"
+            src="https://i.gyazo.com/117b1b0999fd54f99495be9a7e9d7419.png"
+            alt=""
+          />
+          <img
+            className="h-10 w-10 rounded-full"
+            src="https://i.gyazo.com/117b1b0999fd54f99495be9a7e9d7419.png"
+            alt=""
+          />
+          <img
+            className="h-10 w-10 rounded-full"
+            src="https://i.gyazo.com/117b1b0999fd54f99495be9a7e9d7419.png"
+            alt=""
+          />
```

Comment:
worth making a subcomponent for this

Reply:
would you mind elaborating as to why you believe it would be worth it?

Reply:
looks like you're going to repeat this code elsewhere?

Reply:
This is not going to be repeated anywhere else. But this section isnt finished. It needs to have some of the images and words edited.